### ISSUE TITLE
[UPDATE OUTDATED CONTENT] DigitalOcean one-click install page should be deprecated

### ISSUE THREAD

**Issue by philting** [strapi/community-content#445]
**Description**
Documentation still has a full page tutorial that goes over one-click installation with DigitalOcean. Looks like it's just giving a 404 error now.

I also noticed there was a topic here that there were plans to deprecate one-click installation Strapi from DigitalOcean: #118 

It would have been very helpful if the documentation explicitly stated that the one-click installation has been removed from DigitalOcean, in addition to stating your current plans of refactoring manual installation. It wasn't until I dug through the issues here that I found out that this was planned. I think that simply removing the page wouldn't be particularly helpful.

It would also be helpful if the DigitalOcean link pointed to something that said it was deprecated, rather than just throwing a straight 404 error. I don't know how much control over that you have on your end. This is mostly because there have been other tutorials that reference the same Digital Ocean link.

**Link of the content**
https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/installation/digitalocean-one-click.html#creating-a-strapi-project

DigitalOcean URL: https://marketplace.digitalocean.com/apps/strapi

**Comment by ShadaW11** [comment]
Thank you @philting for noting that out.

@pwizla can you help out with this?

**Comment by pwizla** [comment]
Thank you for opening this issue, @philting :-)

Actually, Strapi has never decided to remove anything from DigitalOcean's marketplace. We noticed it a few days ago and are actively working with them on the topic, though we don't have any control over it. 

When we updated the installation guide a few weeks ago, the app was available on DO's marketplace. 

In the meantime, we will soon publish a note on our documentation page to clearly state that the DigitalOcean app is currently offline on their marketplace.
